proposals/0002-root-signature-in-clang.md

```diff
@@ -402,23 +402,75 @@ registers are bound multiple times, or where there are multiple RootFlags
 entries, so subsequent stages should not assume that any given root signature in
 IR is valid.
 
+#### Metadata Schema
 
-> **Open Question**: what should the named metadata be?  There's options I
-> think...
+* RootConstant
+
+| ParameterType | ShaderVisibility | ShaderRegister | RegisterSpace | Num32BitValues |
+|---------------|------------------|----------------|---------------|----------------|
+| String        | int              | int            | int           | int            |
+
+
+* RootDescriptor
+
+| ParameterType | ShaderVisibility | ShaderRegister | RegisterSpace | RootDescriptorFlags |
+|---------------|------------------|----------------|---------------|---------------------|
+| String        | int              | int            | int           | int                 |
+
+
+* DescriptorRange
+
+| RangeType | NumDescriptors | BaseShaderRegister | RegisterSpace | DescriptorRangeFlags |
+|-----------|----------------|--------------------|---------------|----------------------|
+| String    | int            | int                | int           | int                  |
+
+
+* DescriptorTable
```

Comment:
DescriptorTable should have a ShaderVisibility field as well.

Reply:
Also, in the example it seems that DescriptorTable has two fields - the shader visiblity and the reference to the list of descriptor ranges.

Reply:
Updated.